lib/howitzer/meta/base.rb

```diff
@@ -0,0 +1,28 @@
+module Howitzer
+  module Meta
+    # Base class for meta elements, contains utility methods
+    class Base
```

Comment:
👍

Reply:
Looks like a good candidate for Ruby module. Motivation: this class will be never instantiated. 

```ruby
module Howitzer
  module Meta
     module Actions
     end
  end
end
```